src/cursor.jl

```diff
@@ -18,6 +18,14 @@ Base.hash(x::CLCursor) = hash(x.cursor)
 Base.:(==)(c1::CLCursor, c2::CXCursor) = c1.cursor == c2
 Base.:(==)(c1::CXCursor, c2::CLCursor) = c1 == c2.cursor
 
+function Base.convert(::Type{String}, cxstr::CXString)
+    ptr = clang_getCString(cxstr)
+    ptr == C_NULL && return ""
+    s = unsafe_string(ptr)
+    clang_disposeString(cxstr)
+    return s
+end
```

Comment:
This implementation is a bit unsafe. What will happen when someone calls this function twice? The behavior may depend on the implementation of `clang_disposeString`.

This is the implementation of `clang_disposeString`:
```
void clang_disposeString(CXString string) {
  switch ((CXStringFlag) string.private_flags) {
    case CXS_Unmanaged:
      break;
    case CXS_Malloc:
      if (string.data)
        free(const_cast<void *>(string.data));
      break;
    case CXS_StringBuf:
      static_cast<cxstring::CXStringBuf *>(
          const_cast<void *>(string.data))->dispose();
      break;
  }
}
```
It just frees the underlying resources without marking the flag field of the input `CXString` variable. 

So, ...